+++
# Date this page was created.
date = 2021-04-27T00:00:00

# Weight
weight = 1

# Project title.
title = "PhenoMap"

# Project summary to display on homepage.
summary = "Characterize unknown phytoplankton species"

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "phenomap_small.jpg"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["taxonomy", "phytoplankton", "culture", "current_projects"]

# Optional external URL for project (replaces project detail page).
external_link = ""

# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
[header]
image = "phenomap.jpg"
caption = "Eukaryotic tree of phytoplankton"

+++

### Aim
Rapidly accumulating environmental sequencing data have revealed that eukaryotic microbes are far more  diverse and complex than previously thought. However, meta-barcoding and meta-genomics surveys are  severely limited by the fact that the majority of environmental sequences do not match a sequence with  associated phenotypic/taxonomic information in reference databases. Phytoplankton, a polyphyletic group  of  single-celled photosynthetic  organisms  that  play  key  roles  in  aquatic  food  webs  and  global  biogeochemical cycles, comprise an important part of this undescribed diversity. PHENOMAP will address  this major “phenotype gap” for marine phytoplankton by undertaking targeted phenotypic description of  key cryptic lineages. The work plan integrates a suite of state of the art methods that will be applied to two  outstanding existing resources: the Roscoff Culture Collection, which is the largest and most diverse service  collection of living microalgal strains in the world, and the Tara protist sample collection that contains over  15,000  fixed  plankton  samples  from  the  worldwide  Tara  Oceans  expeditions  (2009-2018).  Additional  targeted sampling at 3 French marine stations will complement these resources. Phenotypic analyses will  include  light,  fluorescence  and  electron  microscopy  for  both  live  and  fixed  samples,  as  well  as  photosynthetic  pigment  analysis  for  cultures.  For  fixed  samples,  fluorescent  in-situ  hybridisation  will  be  used to target cells belonging to the most abundant cryptic environmental lineages. Genetic barcoding of  morphologically identified single cells isolated from fixed and live samples will complete the experimental  strategy. We aim to formally describe at least 100 new taxa (including many high taxonomic rank lineages)  and link genotypic to phenotypic information for hundreds of additional existing species, adding significant  value to integrated community databases (PR2, PhytoRef, UniEuk, Ecotaxa) that are increasingly central to  studies on phytoplankton biology, ecology and evolution. Widespread dissemination of scientific outputs  and development of innovative educational and outreach resources is integral to the PHENOMAP approach.

### Partners
* Station Biologique de Roscoff RCC (Ian Probert, Pris Gourvil, Daniel Vaulot)
* Station Biologique de Roscoff UMR7144 (C. de Vargas, S. Romac)
* LERBO, IFREMER, Concarneau (Nicolas Chomerat, K. Mertens)

### Funding
* ANR France
