<!DOCTYPE html>
<html lang="en"><head>
<script src="Microbes-01-origin-eukaryotes_files/libs/clipboard/clipboard.min.js"></script>
<script src="Microbes-01-origin-eukaryotes_files/libs/quarto-html/tabby.min.js"></script>
<script src="Microbes-01-origin-eukaryotes_files/libs/quarto-html/popper.min.js"></script>
<script src="Microbes-01-origin-eukaryotes_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="Microbes-01-origin-eukaryotes_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Microbes-01-origin-eukaryotes_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="Microbes-01-origin-eukaryotes_files/libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="Microbes-01-origin-eukaryotes_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.3.353">

  <title>microbes-01-origin-eukaryotes</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="Microbes-01-origin-eukaryotes_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="Microbes-01-origin-eukaryotes_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="Microbes-01-origin-eukaryotes_files/libs/revealjs/dist/theme/quarto.css">
  <link href="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-titled.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-titled) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-titled.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-titled .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-titled .callout-title  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-titled.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-titled.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-title {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-title {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-title {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-titled .callout-body > .callout-content > :last-child {
    margin-bottom: 0.5rem;
  }

  .callout.callout-titled .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-titled) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-title {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-title {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-title {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-title {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-title {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">


<section id="section" class="slide level2" data-background-image="img/protists/plankton_chronicles.jpg" data-background-size="100%" data-background-opacity="0.3">
<h2></h2>
<p><span class="presenter absolute" style="bottom: 300px; "><strong>Daniel Vaulot and Adriana Lopes dos Santos</strong></span></p>
<p><img data-src="img/institutions/logo-asiame-2023.jpg" class="absolute" style="bottom: 0px; right: 0px; height: 120px; "></p>
<p><img data-src="img/institutions/logo_SBR.png" class="absolute" style="left: 0px; bottom: 0px; height: 70px; "></p>
<p><img data-src="img/institutions/logo-university-of-oslo.png" class="absolute" style="left: 300px; bottom: 0px; height: 70px; "></p>
<p><span class="title absolute" style="bottom: 400px; ">Origin of Eukaryotes</span></p>
<p><span class="absolute smaller" style="bottom: 200px; ">2023-06-19</span></p>
</section>
<section id="outline" class="slide level2 outline-slide" data-background="DarkBlue">
<h2>Outline</h2>
<div>
<ul>
<li class="fragment"><p>Early Life Evolution</p></li>
<li class="fragment"><p>How many domains ?</p></li>
<li class="fragment"><p>The endosymbiotic theory</p></li>
</ul>
</div>
</section>
<section>
<section id="early-life-evolution" class="title-slide slide level1 center" data-background-image="img/microbes/hydrothermal-vent-02.jpg" data-background-size="100%" data-background-opacity="0.5">
<h1>Early Life Evolution</h1>

</section>
<section id="early-history-of-earth" class="slide level2">
<h2>Early History of Earth</h2>
<p><img data-src="img/microbes/earth-history.png" class="absolute" style="top: 50px; right: 0px; width: 600px; "></p>
<div class="columns">
<div class="column" style="width:40%;">
<div>
<ul>
<li class="fragment">Earth is ~ 4.5 billion years old. There is not a single rock that is older than this…</li>
<li class="fragment">Liquid water required for life: first appeared ~ 4.3 billion years ago (volcanic outgassing and icy comets/asteroids).</li>
<li class="fragment">Evidence for life ~ 4.1 billion years ago (carbon isotope ratios, life prefers C<sub>12</sub>)</li>
</ul>
</div>
</div>
</div>
</section>
<section id="parameters-that-limit-life" class="slide level2">
<h2>Parameters that limit life</h2>
<div>
<ul>
<li class="fragment">The laws of chemistry and physics have universal principles which enable us to extrapolate to the conditions under which life would have appeared and could survive elsewhere.</li>
<li class="fragment">These principles suggest that life requires:
<ul>
<li class="fragment">a <strong>liquid</strong> solvent</li>
<li class="fragment">an <strong>energy</strong> source</li>
<li class="fragment"><strong>building blocks</strong> (aas, lipids, nucleotides, etc).</li>
</ul></li>
<li class="fragment">Experimental evidence indicates that organic molecules such as RNA nucleotides, amino acids, and lipids can form <strong>spontaneously</strong> under conditions that were present on the early Earth, providing the preconditions needed for the first living systems (e.g.&nbsp;Oparin, Stanley Miller, Juan Oro, Jeffrey Bada and others… JPL Astrobiology research USA)</li>
<li class="fragment"><strong>Temperature, pH, pressure, salinity,</strong> and <strong>radiation</strong> can influence the availability of <strong>nutrients</strong> and <strong>energy sources</strong>,</li>
<li class="fragment"><strong>Microbial diversity</strong> is structured by these parameters in any environment.</li>
<li class="fragment">Identifying the <strong>minima and maxima</strong> of each parameter and their combined effects, is imperative to advance our understanding of limits of life on Earth and the potential existence of life elsewhere.</li>
</ul>
</div>
</section>
<section id="life-can-adapt-to-a-wide-range-of-parameters" class="slide level2">
<h2>Life can adapt to a wide range of parameters</h2>
<p><img data-src="img/microbes/extreme-environments.jpg" class="absolute" style="bottom: 50px; right: 180px; width: 950px; "></p>

<ul>
<li>Microorganisms (Bacteria and Archaea) have been detected in a variety of extreme environments and virtually in any location where liquid water is available for life to use.</li>
</ul>
<aside><div>
<p>https://www.frontiersin.org/articles/10.3389/fmicb.2019.00780/full</p>
</div></aside></section>
<section id="hyperthermophilic-ancestor-on-going-debate" class="slide level2">
<h2>Hyperthermophilic ancestor? On going debate</h2>
<p><img data-src="img/microbes/tree-LUCA.png" class="absolute" style="top: 50px; right: 0px; width: 550px; "></p>
<div class="columns">
<div class="column" style="width:50%;">
<div>
<ul>
<li class="fragment">The surface of the early Earth is thought to have been much hotter than today. Only superheat-loving microbes (<strong>hyperthermophiles</strong>) would have been able to thrive and survive in such an ‘early times of life’ scenario.</li>
<li class="fragment">Hyperthermophilic archaea and bacteria are found on the deepest, shortest branches of the <strong>SSU rRNA phylogenetic tree</strong> (thick branches)</li>
<li class="fragment">Hyperthermophilic bacteria and archaea often display <strong>chemotrophic</strong> metabolism where compounds like molecular hydrogen, carbon dioxide, sulphur and ferric/ferrous iron are used as electron donors.</li>
</ul>
</div>
</div>
</div>

<aside><div>
<p>LUCA: Last Universal Common Ancestor<br> Stetter, K. O. Hyperthermophiles in the history of life. Philos. Trans. R. Soc. B Biol. Sci. 361, 1837–1842 (2006).</p>
</div></aside></section>
<section id="molecular-adaptations-to-life-at-high-temperature" class="slide level2">
<h2>Molecular adaptations to life at high temperature</h2>
<ul>
<li><strong>RNA</strong> stability: higher GC content</li>
<li><strong>DNA</strong> stability
<ul>
<li>high intracellular solute levels stabilize DNA</li>
<li><strong>reverse DNA gyrase</strong> (found only in hyperthermophiles): introduces positive supercoils into DNA</li>
<li>DNA-binding proteins (histones) compact DNA into nucleosome-like structures</li>
</ul></li>
<li><strong>Protein</strong> folding
<ul>
<li>Chaperones: class of proteins that refold partially denatured proteins</li>
</ul></li>
<li><strong>Lipid</strong> stability: possess dibiphytanyl tetraether type lipids; form a lipid monolayer membrane structure</li>
</ul>
</section>
<section id="reverse-dna-gyrase" class="slide level2">
<h2>Reverse DNA gyrase</h2>
<div class="fragment fade-in-then-out">
<p><img data-src="img/microbes/paper-Catchpole-2019.png" class="absolute" style="top: 0px; right: 0px; width: 550px; "></p>
</div>
<div class="fragment">
<p><img data-src="img/microbes/paper-Catchpole-2019-phylo-conserved-proteins.png" class="absolute" style="left: 0px; bottom: 0px; height: 550px; "></p>
</div>
<div class="fragment">
<p><img data-src="img/microbes/paper-Catchpole-2019-phylo-RG.png" class="absolute" style="bottom: 0px; right: 0px; height: 550px; "></p>
</div>
<div class="fragment">
<p>Results suggest a <strong>nonhyperthermophilic LUCA</strong> and bacterial ancestor, with <strong>hyperthermophily emerging early</strong> in the evolution of the archaeal and bacterial domains.</p>
</div>
</section>
<section id="life-originated-at-hydrothermal-systems-on-ocean-floor" class="slide level2">
<h2>Life originated at hydrothermal systems on ocean floor ?</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div>
<ul>
<li class="fragment">The extremely hot temperatures and levels of ultraviolet radiation at Earth’s surface were likely to be very hostile to the origin of life as we know it.</li>
<li class="fragment">Life may have originated at hydrothermal systems where conditions would have been more stable.</li>
<li class="fragment">Steady and abundant supply of energy (e.g., H<sub>2</sub> and H<sub>2</sub>S) may have been available at these sites.</li>
<li class="fragment">Geochemistry can support abiotic production of molecules required for life (e.g., amino acids, lipids, sugars, and nucleotides), as shown by the experiment before hydrothermal vents were discovered.</li>
</ul>
</div>
</div>
</div>
<p><img data-src="img/microbes/hydrothermal-vent-02.jpg" class="absolute" style="top: 50px; right: 0px; width: 500px; "> <img data-src="img/microbes/hydrothermal_vents_map.svg.png" class="absolute" style="bottom: 0px; right: 20px; width: 450px; "></p>
</section></section>
<section>
<section id="how-many-domains" class="title-slide slide level1 center" data-background-image="img/microbes/loki-god-18e-crop.jpg" data-background-size="100%" data-background-opacity="0.5">
<h1>How many domains</h1>

</section>
<section id="the-three-domains-of-life---carl-woese" class="slide level2">
<h2>The three domains of life - Carl Woese</h2>
<p><img data-src="img/microbes/paper-Woese-1990.png" class="absolute" style="top: 150px; right: 0px; width: 600px; "> <img data-src="img/microbes/woese.jpg" class="absolute" style="top: 0px; right: 0px; width: 200px; "> <img data-src="img/microbes/tree_Woese.png" class="absolute" style="bottom: 0px; right: 0px; width: 700px; "></p>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>Distinctive ribosomal RNAs and tRNAs,</li>
<li>Absence of murein cell walls</li>
<li>Presence of ether-linked lipids in phytanyl chains.</li>
</ul>
<div class="fragment">
<p>Molecular features shared <strong>only between archaea and eukaryotes</strong> appeared consistent with 3 domains hypothesis</p>
<ul>
<li>DNA replication machinery</li>
<li>DNA packed by histones in nucleosome-like structures</li>
<li>Structurally complex RNA polymerases</li>
<li>Translation machinery</li>
</ul>
</div>
</div>
</div>
</section>
<section id="two-vs-three-domains-of-life" class="slide level2">
<h2>Two vs Three domains of life</h2>
<div class="columns">
<div class="column" style="width:60%;">
<ul>
<li>In 1984, based on analysis of rRNA, the evolutionary biologist James Lake (University of California, Los Angeles) proposed that eukaryotes are sisters to archaea that he called eocytes, which means dawn cells. The idea evolved into the two-domain scenario.</li>
</ul>
<div class="fragment fade-out">
<p><img data-src="img/microbes/tree_eocyte.png" class="absolute" style="top: 0px; right: 0px; width: 0px; "> <img data-src="img/microbes/paper-Lake-1988-01.png" class="absolute" style="bottom: 200px; right: 0px; width: 500px; "></p>
</div>
<div class="fragment fade-in-then-out">
<p><img data-src="img/microbes/paper-Lake-1988-02.png" class="absolute" style="bottom: 200px; right: 0px; width: 500px; "></p>
</div>
<div class="fragment">
<ul>
<li>In this evolutionary scenario, Archaea and Bacteria represent the only primary domains of life, and eukaryotes later emerged from lineages within the Archaea domain.<br>
<img data-src="img/microbes/domains_three_vs_two.png" class="absolute" style="bottom: 50px; right: 0px; width: 500px; "></li>
</ul>
</div>
</div>
</div>

<aside><div>
<p>Lake, J. A. Origin of the eukaryotic nucleus determined by rate-invariant analysis of rRNA sequences. Nature 331, 184–186 (1988).</p>
</div></aside></section>
<section id="sampling-the-deep-sea" class="slide level2">
<h2>Sampling the deep-sea</h2>
<p><img data-src="img/microbes/loki-map.png" class="absolute" style="bottom: 50px; right: 0px; width: 400px; "></p>
<p>In 2010, a 2m long gravity core (GC14) was retrieved from the <strong>Arctic</strong> Mid-Ocean Ridge during summer, approximately 15 km northwest of the active venting site Loki’s Castle (between Greenland and Norway), at <strong>3283m</strong> below sea level.</p>
<iframe data-external="1" src="https://www.youtube.com/embed/d_yqE7VlAyw" width="70%" height="70%" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
</section>
<section id="why-loki" class="slide level2">
<h2>Why Loki ?</h2>
<img data-src="img/microbes/loki-god.jpg" class="r-stretch">
<p>Loki is a god in Norse Mythology. He creates trouble. He is elusive and ambiguous.</p>
</section>
<section id="discovery-of-lokiarchaeota---2015" class="slide level2">
<h2>Discovery of Lokiarchaeota - 2015</h2>
<div class="columns">
<div class="column" style="width:60%;">
<div class="fragment">
<ul>
<li>A team of microbiologists, led by <strong>Thijs Ettema</strong> of Uppsala University in Sweden, had been studying since 2010 a subset of Archaea lineages, called the Deep-Sea Archaeal Group (DSAG) or TACK (Thaumarchaeota, Aigarchaeota, Crenarchaeota and Korarchaeota). <img data-src="img/microbes/ettema.jpg" class="absolute" style="top: 0px; right: 0px; width: 200px; "></li>
</ul>
</div>
<div class="fragment">
<ul>
<li>They sequenced DNA extracted directly from Loki samples and assembled genomes exclusively from sequences (Genome-resolved metagenomics)</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>They discovered a new archaea phylum called <strong>Lokiarchaeota</strong> which branches with eukaryotes.</li>
</ul>
<p><img data-src="img/microbes/tree-loki.png" class="absolute" style="top: 200px; right: 0px; width: 500px; "></p>
</div>
</div>
</div>

<aside><div>
<p>Spang, A. et al.&nbsp;Complex archaea that bridge the gap between prokaryotes and eukaryotes. Nature 521, 173–179 (2015).</p>
</div></aside></section>
<section id="metagenome-assembled-genomes" class="slide level2">
<h2>Metagenome assembled genomes</h2>

<p><img data-src="img/microbes/MAG-Meren.png"></p>
<aside><div>
<p>https://merenlab.org/2020/06/27/seminar-series-on-microbial-omics/</p>
</div></aside></section>
<section id="asgard-superphylum" class="slide level2">
<h2>Asgard superphylum</h2>
<p><img data-src="img/microbes/tree-asgard.png" class="absolute" style="top: 150px; right: 0px; width: 500px; "></p>
<div class="columns">
<div class="column" style="width:60%;">
<ul>
<li><p>In 2016 and 2017 Ettema’s team and other researchers announced the discovery of 3 new archaeal phyla related to the Lokis: Thorarchaeota, Odinarchaeota, Heimdallarchaeota.</p></li>
<li><p>The entire grouping of new superphylum was named Asgard after the <strong>realm of the Norse gods</strong>.</p></li>
<li><p>Phylogenomic analyses strongly indicated that the new archaea lineages representing the Asgard superphylum form a monophyletic group with eukaryotes.</p></li>
<li><p>Members of Asgard were found to carry genes encoding proteins that were assumed to be specific to eukaryotes (e.g.&nbsp;actins).</p></li>
</ul>
</div>
</div>

<aside><div>
<p>Zaremba-Niedzwiedzka, K. et al.&nbsp;Asgard archaea illuminate the origin of eukaryotic cellular complexity. Nature 541, 353–358 (2017).</p>
</div></aside></section>
<section id="asgard-superphylum-1" class="slide level2">
<h2>Asgard superphylum</h2>
<p><img data-src="img/microbes/asgard.gif" class="absolute" style="left: 150px; "></p>
</section>
<section id="everybody-was-not-convinced" class="slide level2">
<h2>Everybody was not convinced</h2>

<img data-src="img/microbes/paper-da-cunha-2018.png" class="r-stretch"></section>
<section id="the-final-proof---isolation-of-a-pure-co-culture" class="slide level2">
<h2>The final proof - Isolation of a pure co-culture</h2>
<p><img data-src="img/microbes/paper-Nature-Asgard.png" class="absolute" style="top: 100px; right: 0px; width: 350px; "></p>
<div class="columns">
<div class="column" style="width:70%;">
<div>
<ul>
<li class="fragment"><p>Took long time …. “Through subsequent transfers, we were able (…) to obtain a pure co-culture of the target archaeon MK-D1 and&nbsp;<em>Methanogenium</em>&nbsp;after a <strong>12-year</strong> study—from bioreactor-based pre-enrichment of deep-sea sediments to a final 7 years of <em>in vitro</em> enrichment.”</p></li>
<li class="fragment"><p>Doubling time was estimated to be approximately <strong>14–25&nbsp;days</strong></p></li>
<li class="fragment"><p>The genome encodes <strong>proteins that are only found in Eukarya</strong> (eukaryotic signature proteins): membrane trafficking, vesicle formation and/or transportation, ubiquitin and cytoskeleton formation</p></li>
<li class="fragment"><p>It <strong>catabolizes</strong> ten&nbsp;amino acids and peptides through syntrophic growth with&nbsp;<em>Halodesulfovibrio</em> (sulfate-reducing bacterium)&nbsp;and&nbsp;<em>Methanogenium</em>&nbsp;(methanogenic Archaea)</p></li>
</ul>
</div>
</div>
</div>

<aside><div>
<p>Imachi, H. et al.&nbsp;Isolation of an archaeon at the prokaryote–eukaryote interface. Nature 577, 519–525 (2020)</p>
</div></aside></section>
<section id="candidatus-prometheoarchaeum-syntrophicum" class="slide level2">
<h2><em>Candidatus</em>&nbsp;Prometheoarchaeum syntrophicum</h2>
<p><img data-src="img/microbes/paper-Nature-Asgard-EM.png" class="absolute" style="top: 100px; right: 0px; width: 600px; "></p>
<div class="columns">
<div class="column" style="width:40%;">
<ul>
<li><p>Etymology:</p>
<ul>
<li><strong>Prometheus&nbsp;</strong>(Greek): a Greek god who shaped humans out of mud and gave them the ability to create fire;&nbsp;<strong>archaeum</strong>&nbsp;from&nbsp;archaea&nbsp;(Greek): an ancient life. &nbsp;</li>
<li><strong>Syn</strong>&nbsp;(Greek): together with;&nbsp;<strong>trophy</strong>&nbsp;(Greek) nourish;&nbsp;<strong>icus</strong>&nbsp;(Latin) pertaining to.</li>
</ul></li>
<li><p>The cells produce long branching (g) and straight (h) membrane protrusions</p></li>
<li><p>Cytoplasmatic membrane contains “biphytane”(j) (probably monolayer)</p></li>
</ul>
</div>
</div>
</section>
<section id="an-ever-evolving-story---june-2023" class="slide level2">
<h2>An ever-evolving story - June 2023</h2>
<p><img data-src="img/microbes/paper-Emme-2023.png" class="absolute" style="top: 0px; right: 0px; width: 450px; "></p>
<div class="fragment fade-in-then-out">
<p><img data-src="img/microbes/paper-Emme-2023-tree-02.png" class="absolute" style="top: 100px; left: 0px; height: 500px; "></p>
</div>
<div class="fragment fade-in-then-out">
<p><img data-src="img/microbes/paper-Emme-2023-tree.png" class="absolute" style="top: 100px; left: 0px; height: 500px; "></p>
</div>
<div class="fragment fade-in-then-semi-out">
<p><img data-src="img/microbes/paper-Emme-2023-tree-04.png" class="absolute" style="top: 200px; left: 0px; "></p>
</div>

<aside><div>
<p>Eme, L. et al.&nbsp;Inference and reconstruction of the heimdallarchaeial ancestry of eukaryotes. Nature 1–8 (2023)</p>
</div></aside></section></section>
<section>
<section id="endosymbiotic-origin-of-eukaryotes" class="title-slide slide level1 center" data-background-image="img/microbes/endosymbiosis-secondary-02.png" data-background-size="100%" data-background-opacity="0.3">
<h1>Endosymbiotic Origin of Eukaryotes</h1>

</section>
<section id="earths-history" class="slide level2">
<h2>Earth’s history</h2>
<p><img data-src="img/microbes/earth-history-02.png" class="absolute" style="top: 50px; right: 0px; width: 600px; "></p>
<div class="columns">
<div class="column" style="width:40%;">
<div>
<ul>
<li class="fragment"><strong>LUCA</strong> emerged at least 3.5 - 4 billion years ago.</li>
<li class="fragment">An explosion of <strong>metabolisms</strong> (metabolic diversification) occurred allowing life (microbes) to exploit the various resources available on Earth.</li>
<li class="fragment">Oxygenic <strong>photosynthesis</strong> in the ancestors of cyanobacteria appeared on ~ 3.2 billion years ago</li>
</ul>
</div>
</div>
</div>
</section>
<section id="earths-history-1" class="slide level2">
<h2>Earth’s history</h2>
<p><img data-src="img/microbes/earth-history-02.png" class="absolute" style="top: 50px; right: 0px; width: 600px; "></p>
<div class="columns">
<div class="column" style="width:40%;">
<div>
<ul>
<li class="fragment"><strong>Oxygen</strong> accumulated in the atmosphere lead to the Earth’s first mass extinction (GOT).</li>
<li class="fragment">The rising atmospheric oxygen triggered cellular compartmentalization and eukaryogenesis</li>
<li class="fragment"><strong>LECA</strong> emerged ~1–1.9 billion years ago</li>
<li class="fragment"><strong>Eukaryogenesis</strong> – the emergence of eukaryotic cells – represents a pivotal evolutionary event.</li>
<li class="fragment">The <strong>compartmentalization</strong> of cellular processes allowed the evolution of complex multicellular organisms.</li>
</ul>
</div>
</div>
</div>
</section>
<section id="the-origin-of-the-eukaryotic-cell" class="slide level2">
<h2>The origin of the eukaryotic cell</h2>

<img data-src="img/microbes/plant-cell-annotated.png" class="r-stretch"><p>Most early models envisaged that eukaryotic cells evolved from prokaryotic cells (bacteria) by a process of complexification.</p>
</section>
<section id="the-role-of-symbiosis" class="slide level2">
<h2>The role of symbiosis</h2>
<div class="columns">
<div class="column" style="width:40%;">
<div class="fragment">
<ul>
<li>The idea was originally suggested by <strong>Schimper</strong> in 1883 in a small note. <img data-src="img/microbes/shimper-famintsyn.png" class="absolute" style="top: 0px; right: 0px; width: 500px; "></li>
</ul>
</div>
<div class="fragment">
<ul>
<li><strong>Famintsyn</strong> (1889) worked extensively on algal symbiosis. He also attempted to culture chloroplasts outside the of the host.</li>
</ul>
</div>
<div class="fragment">
<ul>
<li><strong>Merezhkowsky</strong> (1905) remarked the similar structural characteristics of cyanobacteria and chloroplasts. <img data-src="img/microbes/Mereschkowski-young.png" class="absolute" style="bottom: 0px; right: 300px; height: 330px; "> <img data-src="img/microbes/tree-Mereschkowski-1905.png" class="absolute" style="bottom: 0px; right: 0px; height: 350px; "></li>
</ul>
</div>
</div>
</div>
</section>
<section id="lynn-margulis" class="slide level2">
<h2>Lynn Margulis</h2>
<p><img data-src="img/microbes/margulis-02.png" class="absolute" style="top: 200px; right: 0px; width: 400px; "> <img data-src="img/microbes/paper-margulis-67.png" class="absolute" style="top: 0px; right: 0px; width: 350px; "></p>
<div class="columns">
<div class="column" style="width:60%;">
<div>
<ul>
<li class="fragment"><p>In 1967, Lynn Margulis championed the idea of an <strong>endosymbiotic origin of organelles</strong> and further hypothesized that not only mitochondria, chloroplast have symbiotic origin but also cell flagella derived from symbiotic spirochetes (this one she got wrong!).</p></li>
<li class="fragment"><p>She presented a comprehensive symbiotic view of eukaryotic cell evolution based on <strong>cytological, biochemical, and paleontological</strong> evidence in her seminal work “On the origin of mitosing cells”.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="the-endosymbiotic-theory" class="slide level2">
<h2>The endosymbiotic theory</h2>

<img data-src="img/microbes/endosymbiosis.png" class="r-stretch"></section>
<section id="molecular-phylogeny" class="slide level2">
<h2>Molecular phylogeny</h2>
<p>Part of Margulis ideas started to become credible with the advent of molecular phylogeny that allowed demonstrating that conserved genes in <strong>chloroplast</strong> and <strong>mitochondrial</strong> genomes clustered with, respectively, <strong>cyanobacterial</strong> and <strong>alphaproteobacterial</strong> genes in phylogenetic trees.</p>
<p><img data-src="img/microbes/tree-mito-chloro.png" width="600"></p>

<aside><div>
<p>Schwartz, R. M. &amp; Dayhoff, M. O. Origins of Prokaryotes, Eukaryotes, Mitochondria, and Chloroplasts. Science 199, 395–403 (1978).</p>
</div></aside></section>
<section id="chloroplast---primary-endosymbiosis" class="slide level2">
<h2>Chloroplast - Primary endosymbiosis</h2>
<p><img data-src="img/microbes/endosymbiosis-primary.png" class="absolute" style="top: 50px; right: 0px; width: 500px; "></p>
<div class="columns">
<div class="column" style="width:50%;">
<div>
<ul>
<li class="fragment">Photosynthetic eukaryotes emerged from a heterotrophic eukaryote which ate a gram-negative <strong>cyanobacterium</strong> (2 membranes),</li>
<li class="fragment">The cyanobacterium was retained and not digested.</li>
<li class="fragment">The ingested cyanobacterium gradually <strong>integrated</strong> into the cellular machinery as a new organelle: the plastid.</li>
</ul>
</div>
</div>
</div>
</section>
<section id="primary-endosymbiosis-occured-at-least-twice" class="slide level2">
<h2>Primary endosymbiosis occured at least twice</h2>
<p><img data-src="img/microbes/endosymbiosis-primary-02.png" class="absolute" style="bottom: 0px; right: 150px; width: 900px; "></p>
<div class="columns">
<div class="column" style="width:100%;">
<div>
<ul>
<li class="fragment">The first occurred ~ 1.6 billion years ago and gave rise to <strong>glaucophytes</strong>, <strong>red algae</strong> and <strong>green algae</strong> (the ancestors of all plants)</li>
<li class="fragment">The second occurred ~ 90–140 million years ago, establishing a permanent photosynthetic compartment (the chromatophore) in amoebae in the genus&nbsp;<em>Paulinella</em>.&nbsp;</li>
</ul>
</div>
</div>
</div>
</section>
<section id="chloroplast---secondary-endosymbiosis" class="slide level2">
<h2>Chloroplast - Secondary endosymbiosis</h2>
<p><img data-src="img/microbes/endosymbiosis-secondary.png" class="absolute" style="top: 50px; right: 0px; width: 500px; "></p>
<div class="columns">
<div class="column" style="width:50%;">
<div>
<ul>
<li class="fragment">Secondary symbiosis was the process of a <strong>heterotrophic</strong> eukaryote engulfing a green or red algal cell (also eukaryotes).</li>
<li class="fragment">The retained, non-digested cell, became gradually the chloroplast.</li>
</ul>
</div>
</div>
</div>
</section>
<section id="chloroplast---secondary-endosymbiosis-1" class="slide level2">
<h2>Chloroplast - Secondary endosymbiosis</h2>
<p><img data-src="img/microbes/endosymbiosis-secondary-02.png" class="absolute" style="top: 50px; right: 0px; width: 600px; "></p>
<div class="columns">
<div class="column" style="width:50%;">
<div>
<ul>
<li class="fragment">Red, green, and glaucophyte algae (and plants) have tremendous diversity but they still only account for a fraction of the diversity of photosynthetic eukaryotes.</li>
<li class="fragment">Secondary plastids are characterized by additional membranes surrounding the plastid
<ul>
<li class="fragment">Two membranes from primary plastid (cyanobacteria)</li>
<li class="fragment">Cytoplasmic envelope of the primary algal endosymbiont.</li>
<li class="fragment">Phagocytotic vacuole in which the algal prey was taken up.</li>
</ul></li>
</ul>
</div>
</div>
</div>
</section>
<section id="the-nucleomorph" class="slide level2">
<h2>The nucleomorph</h2>
<p><img data-src="img/microbes/endosymbiosis-secondary-03.png" class="absolute" style="top: 50px; right: 0px; width: 700px; "></p>
<div class="columns">
<div class="column" style="width:40%;">
<p><strong>Cryptophytes</strong> and <strong>Chlorarachniophytes</strong> retained a <strong>nucleomorph</strong> which is a remnant of the nucleus of the primitive photosynthetic cell. This is a further evidence of the endosymbiotic theory.</p>
</div>
</div>
</section>
<section id="the-apicoplast" class="slide level2">
<h2>The apicoplast</h2>
<p><img data-src="img/microbes/apicomplexa.png" class="absolute" style="top: 50px; right: 0px; width: 400px; "></p>
<div class="columns">
<div class="column" style="width:60%;">
<div>
<ul>
<li class="fragment"><p><strong>Apicomplexa</strong> are non-photosynthetic successful group of endoparasites of animals, from corals to humans (e.g.&nbsp;<em>Plasmodium</em> causing malaria).</p></li>
<li class="fragment"><p>Evolved from free living algae.</p></li>
<li class="fragment"><p>Contain an <strong>apicoplast</strong> which is a plastid that lost its photosynthetic ability.</p></li>
<li class="fragment"><p>Bound by four membrane. Own genome and expresses a small number of genes.</p></li>
</ul>
</div>
</div>
</div>

<aside><div>
<p>Foth, B. J. &amp; McFadden, G. I. The apicoplast: A plastid in Plasmodium falciparum and other apicomplexan parasites. in International Review of Cytology vol.&nbsp;224 57–110 (Academic Press, 2003).</p>
</div></aside></section></section>
<section>
<section id="protist-diversity" class="title-slide slide level1 center">
<h1>Protist diversity</h1>

<img data-src="img/microbes/tree-plastids-CRAS-2023.jpg" class="r-stretch"></section>
<section id="take-home-messages" class="slide level2 outline-slide" data-background="DarkBlue">
<h2>Take home messages</h2>
<div>
<ul>
<li class="fragment"><p>Origin of eukaryotes is key question</p></li>
<li class="fragment"><p>Discovery of Asgard induce change of paradigm</p></li>
<li class="fragment"><p>Endosymbiosis is key to understand evolution of eukaryotes</p></li>
</ul>
</div>
</section></section>
<section id="questions" class="title-slide slide level1 center" data-background="black">
<h1>Questions ?</h1>
<div class="footer footer-default">

</div>
</section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="Microbes-01-origin-eukaryotes_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'smaller': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: true,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1240,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button', {
        text: function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
        }
      });
      clipboard.on('success', function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      });
      function tippyHover(el, contentFn) {
        const config = {
          allowHTML: true,
          content: contentFn,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start'
        };
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          return note.innerHTML;
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>